android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
Just out of curiosity - gradle-4.x-current is 4.10.3 right now, why move back to 4.7?
React-Native itself has moved to 4.10.2 semi-recently - https://github.com/facebook/react-native/tree/master/gradle/wrapper and 4.10.3 seems to work fine for me?

To be clear, I don't think this is a problem exactly and you may have a good reason (I havent' checked through issues), but if there's no reason then staying current seems better

Reply:
sorry, my bad, will fix it